app/layout.tsx

```diff
@@ -37,7 +38,7 @@ export default async function RootLayout({
       <body className={inter.className}>
         <MantineProvider theme={theme} defaultColorScheme="auto">
           <ModalsProvider>
-            <PublicURLProvider value={process.env.PUBLIC_URL!}>
+            <PublicURLProvider value={env.PUBLIC_URL!}>
```

Comment:
Still got a null assertion here
```suggestion
            <PublicURLProvider value={env.PUBLIC_URL}>
```

Reply:
That one is needed because of how PublicURLProvider works